README.md

```diff
@@ -1,10 +1,123 @@
 # Anbox Cloud NFS operator
 
+## Description
+
 The Anbox Cloud NFS operator charm allows providing additional storage to LXD
 nodes via NFS. This is intended to be used to provide additional shared data
 to Anbox containers, e.g. game assets.
 
+Current features:
+
+* Supports two modes to mount an NFS path, namely `nfs` and `efs`
+* TLS support with EFS
+
+## Usage
+
+### Basic Usage
+
+This operator is a subordinate charm and attaches itself to a principal charm.
+To deploy the charm using its default configuration:
+
+```shell
+juju deploy anbox-cloud-nfs
+```
+
+To mount a basic network path use the following config:
+
+```yaml
+applications:
+  nfs-op:
+    charm: anbox-cloud-nfs
+    channel: stable
+    options:
+      mount_type: nfs
+      nfs_path: <host_address_ip_or_domain>:/
+```
+
+### Use EFS filesystem on AWS without TLS
+
+> Note: This feature has been introduced recently. So if you are
+> running an older version of the charm, please upgrade the charm using
+> `juju refresh anbox-cloud-nfs --channel latest/stable`.
```

Comment:
Sicne the fact we have the charm published to `latest/stable` branch, I feel like we can drop the NOTE completely.

Reply:
yeah makes sense. I kept it for now as upgrade was a step need for older clients.